indexers/vector/FaissIndexer/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from typing import Tuple, Optional
 
+from faiss import normalize_L2
```

Comment:
oh, I just realized, can you move these imports inside the functions using it? Otherwise `jina core` shows a `warning` because FaissIndexer cannot be imported.

Python will only import once, which won't be a problem

Reply:
Yes, I am currently working on this. Unfortunately, now the query gives me not reasonable results. I am working on it.

Reply:
Can you mark this issue as resolved, so that the merge-request can continue?